Orion.Net/Controllers/BaseDataController.cs

```diff
@@ -17,57 +17,28 @@ namespace Orion.Net.Controllers
     public class BaseDataController<T> : Controller where T : ClientScriptResult, new()
     {
         /// <summary>
-        /// Lazy connection to Redis server
+        /// Cache Management of the API
+        /// <list type="table">
+        /// <item><see cref="LocalCache"/> is for a internal cache memory</item>
+        /// <item><see cref="RedisCache"/> is for external cache memory with Redis</item>
+        /// </list>
         /// </summary>
-        internal Lazy<ConnectionMultiplexer> lazyConnection;
-        /// <summary>
-        /// Interface to Redis database for the access to the methods
-        /// </summary>
-        internal IDatabase cacheRedis;
-
-        /// <summary>
-        /// Constructor of <see cref="BaseDataController{T}"/> with the instantiation of the connection to Redis server <see cref="lazyConnection"/> and the database interface <see cref="cacheRedis"/>
-        /// </summary>
-        public BaseDataController()
-        {
-            lazyConnection = new Lazy<ConnectionMultiplexer>(() =>
-            {
-                return ConnectionMultiplexer.Connect("key");
-            });
-            cacheRedis = lazyConnection.Value.GetDatabase(asyncState: true);
-        }
-
-        /// <summary>
-        /// Destructor to dispose the connection to redis server
-        /// </summary>
-        ~BaseDataController()
-        {
-            lazyConnection.Value.Dispose();
-        }
-
+        /// <remarks>Change <see cref="LocalCache"/> to <see cref="RedisCache"/> and vice versa depending you need</remarks>
+        protected ICacheManagement CacheData = new LocalCache();
```

Comment:
Please, set ICacheManagement from constructor by injection instead make a new LocalCache here.
The IOC must be loaded in startup.cs